nautobot_golden_config/forms.py

```diff
@@ -334,29 +334,13 @@ class Meta:
 class GoldenConfigSettingForm(NautobotModelForm):
     """Filter Form for GoldenConfigSettingForm instances."""
 
-    slug = SlugField()
     dynamic_group = utilities_forms.DynamicModelChoiceField(queryset=DynamicGroup.objects.all(), required=False)
 
     class Meta:
         """Filter Form Meta Data for GoldenConfigSettingForm instances."""
 
         model = models.GoldenConfigSetting
-        fields = (
-            "name",
-            "slug",
-            "weight",
-            "description",
-            "backup_repository",
-            "backup_path_template",
-            "intended_repository",
-            "intended_path_template",
-            "jinja_repository",
-            "jinja_path_template",
-            "backup_test_connectivity",
-            "dynamic_group",
-            "sot_agg_query",
-            "tags",
-        )
+        fields = "__all__"
```

Comment:
@jeffkala ok, maybe I fibbed :) I can remove these 3 if they cause us any pain and take care of during 2.0 migration.